posthog/client.py

```diff
@@ -470,6 +484,7 @@ def feature_enabled(
         group_properties={},
         only_evaluate_locally=False,
         send_feature_flag_events=True,
+        geoip_disable=True,
```

Comment:
These defaults should also come from the SDK at the top level?

So, should be `None` default here I'd say, so a user can override it. If it's not `None`, pass on given value below. If it's `None`, use the SDK default (`self.geoip_disable` )